components/scream/src/physics/shoc/shoc_functions_f90.cpp

```diff
@@ -579,6 +583,14 @@ void shoc_diag_second_moments_ubycond(SHOCSecondMomentUbycondData& d)
   d.transpose<ekat::util::TransposeDirection::f2c>();
 }
 
+void shoc_pblintd_cldcheck(SHOCPblintdCldCheckData& d)
+{
+  shoc_init(64, true);
```

Comment:
Shouldn't init use nlev if it's available?

Reply:
yes, changes are made in the new commitment.